### ISSUE TITLE
Bank Module not working - Cata Classic

### ISSUE THREAD

**Issue by kahayes14** [Jaliborc/Bagnon#1994]
**Which software were you running?**
 - Addon version name: 11.0.23
 - Client used: Cata Classic 4.4.1.57564

**Please describe the bug.**
The bank frames do not load, it only shows the vanilla UI. When you attempt to enable it in the frame settings, nothing happens.

**Please describe how to reproduce it.**
Open your personal bank

**Any Screenshots?**
If applicable, please add screenshots to help explain your problem.

**Error Logs are Important!**
No LUA errors are reported for Bagnon

![image_2024-11-22_122522356](https://github.com/user-attachments/assets/7ac93f36-e7cc-4dfe-82f8-1a81d8ae7b7a)

**Comment by mulano1977** [comment]
same problem

**Comment by polischuk16** [comment]
same on classic

**Comment by herkaa** [comment]
same

**Comment by NathanHawks** [comment]
Yeah suddenly no worky this morning. Cata Classic bank frame, can't enable in the options

**Comment by Gordo04** [comment]
I rolled back to version bagnon-11.0.19.zip and it works like it should.  You can find it in the Curseforge app under Versions.

**Comment by mulano1977** [comment]
Addon patched and works now :D thanks
